source/configuration-files/demo_config.h

```diff
@@ -129,7 +129,8 @@ extern void vLoggingPrintf( const char * pcFormatString,
  *
  * #define democonfigMQTT_BROKER_PORT    ( insert here. )
  */
-#define democonfigMQTT_BROKER_PORT        ( 8883 )
+//#define democonfigMQTT_BROKER_PORT        ( 8883 )
+#define democonfigMQTT_BROKER_PORT        ( 1883 )
```

Comment:
This should be reverted, since the demo is still configured to use TLS by default
```suggestion
#define democonfigMQTT_BROKER_PORT        ( 8883 )
```

Reply:
Done.